src/DynamoPackages/PackageManagerExtension.cs

```diff
@@ -151,6 +151,10 @@ public void Ready(ReadyParams sp)
         {
             ReadyParams = sp;
             sp.CurrentWorkspaceChanged += OnCurrentWorkspaceChanged;
+
+            (sp.CurrentWorkspaceModel as WorkspaceModel).CollectingCustomNodePackageDependencies += GetCustomNodePackageFromID;
```

Comment:
@scottmitchell Does this mean newly installed custom node package will be added as well? Maybe not?

Reply:
I think that happens here: https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoPackages/PackageManagerExtension.cs#L255-L269